### PR TITLE
[v7r2] logging changes

### PR DESCRIPTION
**Pull request by chaen**
This PR changes some verbose to debug, and update some calls to use a more recent Logger API

BEGINRELEASENOTES
*Resources
CHANGE: changes some log levels in StorageElement

ENDRELEASENOTES

### CODE REVIEW

File: src/DIRAC/RequestManagementSystem/scripts/dirac_rms_list_req_cache.py [hunk 1]
```diff
@@ -9,6 +9,8 @@
 from __future__ import division
 from __future__ import print_function
 
+from pkg_resources import require
```
Comment (fstagni) [inline src/DIRAC/RequestManagementSystem/scripts/dirac_rms_list_req_cache.py:12]: What is this for?

File: src/DIRAC/Resources/Storage/StorageElement.py [hunk 2]
```diff
@@ -44,6 +44,8 @@
 
 DEFAULT_OCCUPANCY_FILE = 'occupancy.json'
 
+sLog = gLogger.getSubLogger(__name__)
```
Comment (fstagni) [inline src/DIRAC/Resources/Storage/StorageElement.py:47]: is this used?

File: src/DIRAC/RequestManagementSystem/scripts/dirac_rms_list_req_cache.py (hunk 1 above)
Comment (chaen) [reply src/DIRAC/RequestManagementSystem/scripts/dirac_rms_list_req_cache.py:12]: oh, automatic import I suspect....

File: src/DIRAC/Resources/Storage/StorageElement.py (hunk 2 above)
Comment (chaen) [reply src/DIRAC/Resources/Storage/StorageElement.py:47]: Now yes, thanks !

**Review by fstagni — APPROVED** [review]

**Pull request merged**